lib/game/widget_builder.dart

```diff
@@ -231,11 +231,19 @@ class WidgetBuilder {
       ),
     );
 
-    if (_hasBasicGestureDetectors(game)) {
+    final hasBasicDetectors = _hasBasicGestureDetectors(game);
+    final hasAdvancedDetectors = _hasAdvancedGesturesDetectors(game);
+
+    if (hasBasicDetectors && hasAdvancedDetectors) {
+      print(
```

Comment:
I wonder if there is any better solution for logging than just a print

Reply:
Couldn't this be an `assert`?
Or do we not want to crash in this case?

Reply:
Yeah, it would crash. It is an option as well, but I think I prefer this to be a warning.

Reply:
A difference between this and assert is that asserts are removed from the code on release mode.

I vote for asserts here since it is an API disambiguation.

Reply:
agree with assert, ideally should be a compile time error, but if we can't, assert is the best next thing

Reply:
Done the assert!